code/clue.js

```diff
@@ -1,59 +1,284 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
+// OBJECTS FOR ALL THE SUSPECTS
 const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
+  firstName: "Jacob",
+  lastName: "Green",
+  color: "#008000", //green
+  description: "He has a lot of connections",
   age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  image: "assets/green.png",
+  occupation: "Entrepreneur",
+  favouriteWeapon: "poisoN",
+};
+
+const mrsWhite = {
+  firstName: "Elisabeth",
+  lastName: "Pale",
+  color: "#F5DEB3", //wheat
+  description: "She loves watering plants and drinking tea",
+  age: 67,
+  image: "assets/white.png",
+  occupation: "Retired firewoman",
+  favouriteWeapon: "ropE",
+};
+
+const profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "#dda0dd", //plum
+  description: "Easily intimidated but not by differential equations.",
+  age: 42,
+  image: "assets/plum.png",
+  occupation: "Professor of engineering",
+  favouriteWeapon: "baT",
+};
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "#ff4500", //orangered
+  description:
+    "She is a lady in red, easily remembered by her brown curly hair. Not a small talker",
+  age: 34,
+  image: "assets/scarlet.png",
+  occupation: "Theoretical physicist",
+  favouriteWeapon: "candlesticK",
+};
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "#7924dc", //purpleblue
+  description: "An elegant and mysterious lady. Rarely speaks to anyone.",
```

Comment:
So I see that you have a description for every suspect object, but not  in the function & which shows on your website's killer-cards. Maybe delete the description in every object in the JS? Or add: document.getElementById("killerDescription").innerHTML = `${mysteryObject.killer.Description}`; to the function pickKiller... :)